cql2pgjson/src/test/java/org/folio/cql2pgjson/CompoundIndexIT.java

```diff
@@ -50,22 +54,54 @@ public void runSetup() throws Exception {
       throw new RuntimeException(e);
     }
   }
+  private List<String> cqlc(String cql) {
+    try {
+      String sql = cql2pgJsonTablec.toSql(cql).toString();
+      return firstColumn("select jsonb->>'user' from tablec " + sql);
+    } catch (QueryValidationException e) {
+      throw new RuntimeException(e);
+    }
+  }
+  private List<String> cqld(String cql) {
+    try {
+      String sql = cql2pgJsonTabled.toSql(cql).toString();
+      return firstColumn("select jsonb->>'user' from tabled " + sql);
+    } catch (QueryValidationException e) {
+      throw new RuntimeException(e);
+    }
+  }
+//  @Test
+//  public void compoundIndexTestMultiFieldNames() throws Exception {
+//    assertThat(cqla("fullname == Tom Jones"), containsInAnyOrder("Tom"));
+//  }
+//
+//  @Test
+//  public void compoundIndexTestMultiFieldNames2() throws Exception {
+//    assertThat( cqlb("address == Boston MA"),  containsInAnyOrder("first0"));
+//  }
+//  @Test
+//  public void compoundIndexTestMultiFieldNames3() throws Exception {
+//    assertThat(cqla("ftfield = first0 last0"), containsInAnyOrder("Mike"));
+//  }
+//
+//  @Test
+//  public void compoundIndexTestMultiFieldNames4() throws Exception {
+//    assertThat( cqlb("ftfield = first1 last1"),  containsInAnyOrder("first1"));
+//  }
```

Comment:
Can you remove this commented out code? It can be retrieved from Git history.

Reply:
This was committed as commented out by accident, fixing mistake